README.md

```diff
@@ -56,13 +56,16 @@
 
 - **Expense Tracking**: Keep a record of your expenses to monitor and manage your spending habits effectively.
 
-- **Budget Planning**: Create budgets for various categories and track your spending against those budgets.
+- **Budget Planning**: Create budgets for various Groups and track your spending against those budgets.
 
 - **Transaction Analysis**: Gain insights into your financial patterns with detailed transaction categorization and analysis.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
 
+- [Live Demo Link](https://budgetapp-h8vp.onrender.com)
+- [Live Demo Video](https://www.loom.com/share/54ee61f0ee2c4fd484b148f76c9d2e5f?sid=82bac7a6-300a-428c-8e53-154b4bdadab8)
```

Comment:
- [ ]  Hi, you did a great job. Can you record a new video? because you missed highlighting a piece of code that you found interesting or are proud of making it. It was mentioned from the previous review and it is part of the requirements.